refinery/tool_manager/models.py

```diff
@@ -413,6 +417,10 @@ def launch(self):
         launched container's url
             - <HttpResponseBadRequest>, <HttpServerError>
         """
+        client = DockerClientWrapper()
+        max = settings.DJANGO_DOCKER_ENGINE_MAX_CONTAINERS
```

Comment:
could be confusing with Python's `max()` builtin?

Reply:
good thought